src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import core.basesyntax.models.Circle;
+import core.basesyntax.models.IsoscelesTrapezoid;
+import core.basesyntax.models.Rectangle;
+import core.basesyntax.models.RightTriangle;
+import core.basesyntax.models.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final String DEFAULT_COLOR = "rgb(255, 255, 255)";
+    public static final double DEFAULT_RADIUS = 10.0;
```

Comment:
Consider replacing the magic number `10.0` with a named constant to improve code readability and maintainability. This aligns with the checklist requirement to avoid magic numbers in your code.